pkg/playground/react-patterns.js

```diff
@@ -22,8 +22,8 @@ require([
     "react",
     "performance/dialog-view",
     "playground/react-demo-dialog",
-    "base1/cockpit-components",
-], function(cockpit, React, dialog_pattern, demo_dialog, components) {
+    "base1/cockpit-components-terminal",
+], function(cockpit, React, dialog_pattern, demo_dialog, cockpit_terminal) {
```

Comment:
Hm. The idea was that we have all components in the same namespace. Is there no way to do that with jsmodule?